src/ansys/fluent/core/__main__.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+
+from ansys.fluent.core.launcher.launcher import launch_fluent
+from ansys.fluent.core.session_solver import Solver
+
+
+def setup_for_fluent(version: str, mode: str):
+    """Uses global PyConsole objects."""
+    session = launch_fluent(version=version, mode=mode)
+    if mode == "meshing":
+        globals()["meshing"] = session
```

Comment:
Is this quite right? On Fluent meshing console side, `meshing` is the meshing state engine which is equivalent to `session.meshing`.

Reply:
Done. Thank you.